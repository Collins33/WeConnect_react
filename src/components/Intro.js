import React, { Component } from 'react';
import {Animated} from 'react-animated-css';
import '../App.css';

class Intro extends Component {
  state = {

  }
  render() {
    return (
      <Animated animationIn="zoomIn" animationOut="fadeOut" isVisible={true}>
        <div className="row intro">
          <h1 className="text-center intro-text title">WeConnect</h1>
          <h2 className="text-center intro-text">
            A new way to connect with businesses around you
          </h2>
          <a href="/businesses"><button  class="btn btn-dark intro-button">EXPLORE</button></a>
        </div>
      </Animated>

    );
  }
}

export default Intro;
