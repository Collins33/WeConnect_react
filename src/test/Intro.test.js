import Intro from '../components/Intro';

describe('landing page renders', () => {
     const wrapper = shallow(<Intro />);
     it('shows the website logo', () => {
          expect(shallow(<Intro />).find('h1.text-center').exists()).toBe(true);
     });
     it('renders an explore button', () => {
          expect(shallow(<Intro />).find('button').exists()).toBe(true);
     });
     it('renders business update form', () => {
          expect(shallow(<Intro />)).toMatchSnapshot();
     });
});
