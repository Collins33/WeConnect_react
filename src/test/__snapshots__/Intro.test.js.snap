// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing page renders renders business update form 1`] = `
<Animated
  animateOnMount={true}
  animationIn="zoomIn"
  animationInDelay={0}
  animationOut="fadeOut"
  animationOutDelay={0}
  className=""
  isVisible={true}
  style={Object {}}
>
  <div
    className="row intro"
  >
    <h1
      className="text-center intro-text title"
    >
      WeConnect
    </h1>
    <h2
      className="text-center intro-text"
    >
      A new way to connect with businesses around you
    </h2>
    <a
      href="/businesses"
    >
      <button
        class="btn btn-dark intro-button"
      >
        EXPLORE
      </button>
    </a>
  </div>
</Animated>
`;
